responses/03_workflow-steps.md

```diff
@@ -141,7 +141,7 @@ jobs:
           password: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
 
       - name: Deploy web app container
-        uses: azure/webapps-container-deploy@v1
+        uses: azure/webapps-container-deploy@v2
```

Comment:
This is using the wrong action.  This action has been deprecated. This line should be changed to:

uses: azure/webapps-deploy@v2

Everything else can stay the same